raiden-ts/src/epics.ts

```diff
@@ -1,25 +1,56 @@
-import { Observable, from, of } from 'rxjs';
+import { Observable, from, of, combineLatest } from 'rxjs';
 import {
   catchError,
   filter,
   mergeMap,
   takeWhile,
   takeUntil,
   pluck,
-  distinctUntilChanged,
+  startWith,
+  map,
 } from 'rxjs/operators';
 import { isActionOf } from 'typesafe-actions';
 import { negate } from 'lodash';
 
 import { RaidenState } from './state';
 import { RaidenEpicDeps } from './types';
 import { RaidenAction, raidenShutdown } from './actions';
+import { getPresences$ } from './transport/utils';
+import { pfsListUpdated } from './path/actions';
+import { Address } from './utils/types';
 
 import * as ChannelsEpics from './channels/epics';
 import * as TransportEpics from './transport/epics';
 import * as TransfersEpics from './transfers/epics';
 import * as PathFindEpics from './path/epics';
 
+/**
+ * This function maps cached/latest relevant values from action$ & state$
+ *
+ * @param action$ - Observable of RaidenActions
+ * @param state$ - Observable of RaidenStates
+ * @returns NEVER, to watch and dispose subject's subscription
+ */
+const getLatest$ = (action$: Observable<RaidenAction>, state$: Observable<RaidenState>) =>
+  combineLatest(
+    action$,
```

Comment:
`combineLatest` is giving some deprecation warnings for a while now. Checking the JSDoc I see the following:
/** @deprecated Pass arguments in a single array instead `combineLatest([a, b, c])` */